### ISSUE TITLE
Demo "10. Modulation" is broken

### ISSUE THREAD

**Issue by rahji** [gibber-cc/gibber#91]
Trying execute the first two commands:

```
b = Box().render()
b.size = lfo( 'sine', .5, .5, 1 )
```

fails with the errors: 

```
ReferenceError: storepos is not defined
ReferenceError: b is not defined
Cannot read properties of undefined (reading 'size')
```

**Comment by rahji** [comment]
Same for Intro to CSG demo

**Comment by charlieroberts** [comment]
Fixed in 1fe283e thanks!
